core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -242,7 +223,7 @@ tableElement
     ;
 
 columnDefinition
-    : qualifiedName type (DEFAULT literal)? (NOT NULL)? (COMMENT string)? (WITH properties)?
+    : qualifiedName type (NOT NULL)? (PRIMARY KEY)? (COMMENT string)? (WITH properties)?
```

Comment:
I think this 
```sql
CREATE TABLE (
  foo t primary key
)
```

might be just a syntax sugar for 
```sql
CREATE TABLE (
  foo t,
  PRIMARY KEY (foo)
)
```

this matters esp when you want your primary key to span 2+ columns